packages/agoric-cli/test/agops-oracle-smoketest.sh

```diff
@@ -25,24 +25,27 @@ fi
 set -x
 
 # this is in economy-template.json in the oracleAddresses list (agoric1dy0yegdsev4xvce3dx7zrz2ad9pesf5svzud6y)
-# to use it run `agd keys oracle2 --interactive` and enter this mnenomic:
+# to use it run `agd keys --keyring-backend=test add oracle2 --interactive` and enter this mnenomic:
 # dizzy scale gentle good play scene certain acquire approve alarm retreat recycle inch journey fitness grass minimum learn funny way unlock what buzz upon
 WALLET2=oracle2
 
+WALLET_BECH32=$(agd keys show $WALLET --keyring-backend=test --output json | jq -r .address)
+WALLET2_BECH32=$(agd keys show $WALLET2 --keyring-backend=test --output json | jq -r .address)
```

Comment:
the commands take names, right? let's not use Bech32 when we don't have to; it's harder to tell which is which

Reply:
We need to use BECH32 for `agoric wallet send` because if you just pass the name, it looks in the os keyring for the BECH32 which doesn't work on WSL. Directly providing the BECH32 bypasses that step.

Reply:
`--keyring-backend=test` should allow the lookup to work. If it doesn't, we should fix the agoric CLI like I did for the other commands so it does.